packages/lesswrong/server/exportUserData.ts

```diff
@@ -45,14 +60,30 @@ export const exportUserData = async (
   }
 
   const userId = user._id;
+  const email = user.email;
 
   const entries: Entry<CollectionNameString>[] = [
     [Users, {fetch: () => Promise.resolve([user])}],
     [Bans, Bans.find({userId})],
+    [CkEditorUserSessions, CkEditorUserSessions.find({userId})],
     [ClientIds, ClientIds.find({userIds: userId})],
     [Collections, Collections.find({userId})],
     [Comments, Comments.find({userId})],
     [Conversations, Conversations.find({participantIds: userId})],
+    [CurationEmails, CurationEmails.find({userId})],
+    [DialogueChecks, DialogueChecks.find({userId})],
+    [GardenCodes, GardenCodes.find({userId})],
+    [GoogleServiceAccountSessions, GoogleServiceAccountSessions.find({email})],
+    [ElectionVotes, ElectionVotes.find({userId})],
+    [ElicitQuestionPredictions, ElicitQuestionPredictions.find({userId})],
+    [PostRecommendations, PostRecommendations.find({userId})],
+    [RecommendationsCaches, RecommendationsCaches.find({userId})],
+    [ReviewVotes, ReviewVotes.find({userId})],
+    [TypingIndicators, TypingIndicators.find({userId})],
+    [UserEAGDetails, UserEAGDetails.find({userId})],
+    [UserJobAds, UserJobAds.find({userId})],
+    [UserRateLimits, UserRateLimits.find({userId})],
+    [LWEvents, LWEvents.find({userId})],
```

Comment:
This might be _very_ large for some users, fwiw.

Reply:
Yeah, that's true, but if the main purpose of this script is GDPR requests then it's required.